scripts/gen-openapi.sh

```diff
@@ -0,0 +1,11 @@
+# Download Snowbounds published API and fix it up locally
+tmpfile=$(mktemp /tmp/openapi.XXXXXX)
```

Comment:
```sh
tmpfile="$(mktemp /tmp/openapi.XXXXXX)"

function cleanup() {
    rm -rf "${tmpfile}"
}

trap EXIT cleanup
```

Reply:
are you really worried that `mktemp` will produce a filename with a space in it?

Reply:
Rule number one of bash - never trust anything :) In this context, you never have to wonder - why is it valid for _this_ variable to be unquoted? Simply not a useful distinction